zuul.d/arista-eos-jobs.yaml

```diff
@@ -69,213 +49,69 @@
+    nodeset: eos-4.24.6-python38
     vars:
-      ansible_test_do_number: 2
+      test_ansible_python_interpreter: /usr/bin/python3.8
+      ansible_test_integration_targets: "eos_.*"
+      ansible_test_fips_mode: true
 
 - job:
-    name: ansible-test-network-integration-eos-network_cli-libssh-python36-stable211-scenario01
-    parent: ansible-test-network-integration-eos-network_cli-python36-stable211
+    name: network-ee-integration-tests-eos-libssh-stable-2.11
+    parent: network-ee-integration-tests-eos-stable-2.11
     vars:
+      test_ansible_python_interpreter: /usr/bin/python3.8
       ansible_test_network_cli_ssh_type: libssh
-      ansible_test_do_number: 1
+      ansible_test_skip_tags: local,httpapi
 
-- job:
-    name: ansible-test-network-integration-eos-network_cli-libssh-python36-stable211-scenario02
-    parent: ansible-test-network-integration-eos-network_cli-python36-stable211
-    vars:
-      ansible_test_network_cli_ssh_type: libssh
-      ansible_test_do_number: 2
+# =============================================================================
 
 - job:
-    name: ansible-test-network-integration-eos-httpapi-python36-stable29
-    parent: ansible-test-network-integration-eos-httpapi
-    nodeset: eos-4.24.6-python36
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
-    vars:
-      ansible_test_python: 3.6
+    name: network-ee-integration-tests-eos-stable-2.9
+    parent: ansible-network-eos-appliance
 
 - job:
-    name: ansible-test-network-integration-eos-network_cli-python36-stable29
-    parent: ansible-test-network-integration-eos-network_cli
-    nodeset: eos-4.24.6-python36
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
+    name: network-ee-integration-tests-eos-stable-2.9
+    parent: network-ee-integration-tests-stable-2.9
+    nodeset: eos-4.24.6-python38
     vars:
-      ansible_test_python: 3.6
+      test_ansible_python_interpreter: /usr/bin/python3.8
+      ansible_test_integration_targets: "eos_.*"
+      ansible_test_fips_mode: true
 
 - job:
-    name: ansible-test-network-integration-eos-network_cli-libssh-python36-stable29
-    parent: ansible-test-network-integration-eos-network_cli-python36-stable29
+    name: network-ee-integration-tests-eos-libssh-stable-2.9
+    parent: network-ee-integration-tests-eos-stable-2.9
     vars:
+      test_ansible_python_interpreter: /usr/bin/python3.8
       ansible_test_network_cli_ssh_type: libssh
-
-- job:
-    name: ansible-test-network-integration-eos-python37
-    parent: ansible-test-network-integration-eos
-    nodeset: eos-4.24.6-python37
-    vars:
-      ansible_test_python: 3.7
-
-- job:
-    name: ansible-test-network-integration-eos-python38
-    parent: ansible-test-network-integration-eos
-    nodeset: eos-4.24.6-python38
-    vars:
-      ansible_test_python: 3.8
+      ansible_test_skip_tags: local,httpapi
```

Comment:
this should be test_ansible_skip_tags.

Also, we should be safe to drop 'local' here, give we stopped testing local connection. However, you may want to double check ansible_connection=local is removed from arista.eos jobs.